config/routes.rb

```diff
@@ -259,7 +259,7 @@
     # Keeping the urls to old guidance urls, but they need to lead to new api-reference ones
     get "/guidance/home", to: redirect("/api-reference")
     get "/guidance/ecf-usage", to: redirect("/api-reference/ecf-usage")
-    get "/guidance/npq-usage", to: redirect("/api-reference/npq-usage")
+    get "/guidance/npq-usage", to: redirect("/api-reference/npq-usage"), constraints: NpqApiEndpoint
```

Comment:
For this type of constraint, i created a request spec to make sure its removed:
https://github.com/DFE-Digital/early-careers-framework/pull/5237/files#diff-290c429c7303749f1b5ef4b8a375f7e24f43d90c12adfc98e819804adc277050R19

In this case you can have route not exist when feature active, and redirect when feature not active.